src/main/java/ch/njol/skript/events/EvtItem.java

```diff
@@ -51,46 +51,50 @@
 				.since("<i>unknown</i> (before 2.1)");
 		Skript.registerEvent("Item Spawn", EvtItem.class, ItemSpawnEvent.class, "item spawn[ing] [[of] %itemtypes%]")
 				.description("Called whenever an item stack is spawned in a world, e.g. as drop of a block or mob, a player throwing items out of his inventory, or a dispenser dispensing an item (not shooting it).")
-				.examples("")
+				.examples("on item spawn of iron sword:",
+						"\tbroadcast \"Someone dropped an iron sword!\"")
 				.since("<i>unknown</i> (before 2.1)");
 		Skript.registerEvent("Drop", EvtItem.class, PlayerDropItemEvent.class, "[player] drop[ing] [[of] %itemtypes%]")
 				.description("Called when a player drops an item from his inventory.")
-				.examples("")
+				.examples("on drop:")
 				.since("<i>unknown</i> (before 2.1)");
 		// TODO limit to InventoryAction.PICKUP_* and similar (e.g. COLLECT_TO_CURSOR)
 		Skript.registerEvent("Craft", EvtItem.class, CraftItemEvent.class, "[player] craft[ing] [[of] %itemtypes%]")
 				.description("Called when a player crafts an item.")
-				.examples("")
+				.examples("on craft:")
 				.since("<i>unknown</i> (before 2.1)");
 		if (hasPrepareCraftEvent) {
 			Skript.registerEvent("Prepare Craft", EvtItem.class, PrepareItemCraftEvent.class, "[player] (preparing|beginning) craft[ing] [[of] %itemtypes%]")
 					.description("Called just before displaying crafting result to player. Note that setting the result item might or might not work due to Bukkit bugs.")
-					.examples("")
+					.examples("on preparing craft of torch:")
 					.since("2.2-Fixes-V10");
 		}
 		Skript.registerEvent("Pick Up", EvtItem.class, PlayerPickupItemEvent.class, "[player] (pick[ ]up|picking up) [[of] %itemtypes%]")
 				.description("Called when a player picks up an item. Please note that the item is still on the ground when this event is called.")
-				.examples("")
+				.examples("on pick up:")
 				.since("<i>unknown</i> (before 2.1)");
 		// TODO brew event
 //		Skript.registerEvent("Brew", EvtItem.class, BrewEvent.class, "brew[ing] [[of] %itemtypes%]")
 //				.description("Called when a potion finished brewing.")
-//				.examples("")
+//				.examples("on brew")
 //				.since("2.0");
 		if (hasConsumeEvent) {
 			Skript.registerEvent("Consume", EvtItem.class, PlayerItemConsumeEvent.class, "[player] ((eat|drink)[ing]|consum(e|ing)) [[of] %itemtypes%]")
 					.description("Called when a player is done eating/drinking something, e.g. an apple, bread, meat, milk or a potion.")
-					.examples("")
+					.examples("on consume:")
 					.since("2.0");
 		}
-		
 		Skript.registerEvent("Inventory Click", EvtItem.class, InventoryClickEvent.class, "[player] inventory(-| )click[ing] [[at] %itemtypes%]")
 				.description("Called when clicking on inventory slot.")
-				.examples("")
+				.examples("on inventory click:",
+						"\tif event-item is stone:",
+						"\t\tgive player 1 stone",
+						"\t\tremove 20$ from player's balance")
```

Comment:
Just making sure - it really works with the `$`?

Reply:
yes money type